app/src/main/java/org/gdg/frisbee/android/api/FirebaseDynamicLinksHub.java

```diff
@@ -0,0 +1,22 @@
+package org.gdg.frisbee.android.api;
+
+import org.gdg.frisbee.android.api.model.FirebaseDynamicLinksResponse;
+
+import okhttp3.RequestBody;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.Field;
+import retrofit2.http.FormUrlEncoded;
+import retrofit2.http.POST;
+import retrofit2.http.Query;
+
+/**
+ * Created by unstablebrainiac on 26/5/17.
+ */
+
+public interface FirebaseDynamicLinksHub {
+
+    @FormUrlEncoded
+    @POST("/v1/shortLinks")
+    Call<FirebaseDynamicLinksResponse> getShortenedUrl(@Query("key") String api_key, @Body RequestBody body);
```

Comment:
Just naming: Can you rename `api_key` to `apiKey`. We try not to use snail case. 

Also what about `shortenUrl` as a method name?